flight/tests/server.rs

```diff
@@ -149,60 +157,235 @@ fn events(start_index: u64) -> Vec<ConclusionEvent> {
+        +-------+-------------------------------------------------------------+-------------+-------------+---------------------------------------------------------+-------------------------------------------------------------+------------+---------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------+
+        | 43    | baeabeif2fdfqe2hu6ugmvgozkk3bbp5cqi4udp5rerjmz4pdgbzf3fvobu | 3           | did:key:bob | {controller: 6469643a6b65793a626f62, model: 6d6f64656c} | baeabeihyzbu2wxx4yj37mozb76gkxln2dt5zxxasivhuzbnxiqd5w4xygq | 1          |                                                                                 | [baeabeials2i6o2ppkj55kfbh7r2fzc73r2esohqfivekpag553lyc7f6bi]                                                              |
+        | 44    | baeabeif2fdfqe2hu6ugmvgozkk3bbp5cqi4udp5rerjmz4pdgbzf3fvobu | 3           | did:key:bob | {controller: 6469643a6b65793a626f62, model: 6d6f64656c} | baeabeifwi4ddwafoqe6htkx3g5gtjz5adapj366w6mraut4imk2ljwu3du | 0          | {"metadata":{"foo":true},"content":[{"op":"replace", "path": "/a", "value":1}]} | [baeabeihyzbu2wxx4yj37mozb76gkxln2dt5zxxasivhuzbnxiqd5w4xygq, baeabeials2i6o2ppkj55kfbh7r2fzc73r2esohqfivekpag553lyc7f6bi] |
+        +-------+-------------------------------------------------------------+-------------+-------------+---------------------------------------------------------+-------------------------------------------------------------+------------+---------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------+"#]].assert_eq(&formatted);
+    Ok(())
+}
+
+#[test(tokio::test)]
+async fn conclusion_events_stream() -> Result<()> {
+    let mut feed = MockFeed::new();
+    feed.expect_max_highwater_mark()
+        .once()
+        .return_once(|| Ok(None));
+    feed.expect_conclusion_events_since()
+        .returning(|h, l| Ok(events(1, h as u64, l as usize)));
+
+    let (mut client, _ctx) = start_server(feed).await;
+
+    let info = client
+        .execute(
+            r#"
+            SELECT
+                "index",
+                cid_string(stream_cid) as stream_cid,
+                stream_type,
+                controller,
+                dimensions,
+                cid_string(event_cid) as event_cid,
+                event_type,
+                data::varchar as data,
+                array_cid_string(previous) as previous
+            FROM conclusion_events_stream LIMIT 2"#
```

Comment:
Here is a test that shows you can query the _stream table to get the data. Of note in this test is there is no logic to wait till the test harness has written the data, as that _is_ the feature of the stream table. It is a blocking query until the limit is reached. The non stream table queries have explicit logic to wait until the test harness knows the data is written before querying.